### PR TITLE
Fix: tools - Fix yaml loader, improve banner

### PR DESCRIPTION
**Pull request by johnnykeats**
Prevents yaml.loader warning
Better banner

### CODE REVIEW

File: bootset/bootset [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # ╚═════╝ ╚══════╝ ╚═════╝ ╚══════╝╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═══╝ ╚══▀▀═╝  ╚═════╝ ╚═╝╚══════╝╚══════╝
 #
 # booset tool, to manage nodes PXE boot
-# 2019_2020 - Benoît Leveugle <benoit.leveugle@sphenisc.com>
+# 2019_2021 - Benoît Leveugle <benoit.leveugle@sphenisc.com>
```
Comment (oxedions) [inline bootset/bootset]: I don't think we should keep the dates in the tool. This is not a changelog, so this is not a valuable data.

**Review by oxedions — COMMENTED** [review]
I like this PR, since it gets ride of the warning, and indeed we should have a small banner.

I added a comment 🙂

File: bootset/bootset (hunk 1 above)
Comment (johnnykeats) [reply bootset/bootset]: OK. Updated using users registered in github history on this file at https://github.com/bluebanquise/bluebanquise/blob/master/roles/core/pxe_stack/files/bootset.py

Comment (oxedions) [reply bootset/bootset]: Perfect ! Bruno's name is Travouillon. Once this is done, ok for me to merge !

**Pull request merged**